aibrary/resources/aibrary.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
it's better to rename this file as `aibrary` is the name of the package and there would be issues with imports if we are not careful.

Reply:
@CenaAshoori